lib/committee/middleware/response_validation.rb

```diff
@@ -38,7 +38,7 @@ def handle(request)
 
       class << self
         def validate?(status, validate_success_only)
-          status != 204 && (!validate_success_only || (200...300).include?(status))
+          !([204, 304].include?(status)) && (!validate_success_only || (200...300).include?(status))
```

Comment:
複雑になってきているので、case文にする方が分かりやすいかも
```suggestion
          case status
          when 204
            false
          when 200..299
            true
          when 304
            false
          else
            !validate_success_only
          end
```